components/loginscreen/Loginscreen.tsx

```diff
@@ -8,7 +8,7 @@ function Logo() {
     <div className={styles.background}>
       <h1 className={styles.headline}>Meet Albert</h1>
       <Input />
-      <Button label="Discover" />
+      <Button label="Discover" link={"/introduce"} />
```

Comment:
I'm not pretty sure but I remember something that we are supposed to use the <Link element or am I mixing something up??